### ISSUE TITLE
Problems with firefox 65

### ISSUE THREAD

**Issue by sajaysurya** [tridactyl/tridactyl#1319]
*   Brief description of the problem: While trying to open a new tab by pressing "t" and typing the new site address, the address bar is instead activated and tridactyl completely stops working. Unfortunately couldn't disable tridactyl as the addons page in firefox is also not working. 

*   Steps to reproduce:

    1.  open firefox and press "t"
    2.  enter a site address and press enter key 
    3.  try to use tridactyl shortcuts
    4.  try opening the firefox addons page

*   Tridactyl version (`:version`): 1.14.6

*   Firefox version (Top right menu > Help > About Firefox): 65.0

*   URL of the website the bug happens on: any site

*   Config (in a new tab, run `:viewconfig`, copy the url and paste it somewhere like gist.github.com): https://gist.github.com/sajaysurya/883e5cedeede244ece30723fc0952858

*   Contents of ~/.tridactylrc or ~/.config/tridactyl/tridactylrc (if they exist):

```
" Place this at $XDG_CONFIG_DIR/tridactyl/tridactylrc (that's
" ~/.config/tridactyl/tridactylrc to mere mortals) or ~/.tridactylrc and
" install the native messenger (:installnative in Tridactyl). Run :source to
" get it in the browser, or just restart.

" NB: If you want "vim-like" behaviour where removing a line from 
" here makes the setting disappear, uncomment the line below.

"sanitise tridactyllocal tridactylsync

"
" Binds
"
bind J tabprev
bind K tabnext

"
" GUI changes
"
guiset_quiet gui none

" 
" Custom Search Engines
"
set searchurls.ddg https://duckduckgo.com/?q=
set searchurls.azn https://www.amazon.co.uk/s/ref=nb_sb_noss?url=search-alias%3Daps&field-keywords=
set searchurls.dic https://www.vocabulary.com/dictionary/
"
" Misc settings
"
set searchengine google
set allowautofocus false
set theme dark

"
" Blacklists
"
blacklistadd localhost

" This will have to do until someone writes us a nice syntax file :)
" vim: set filetype=vim:
```

*   Operating system: linux, ubuntu 18.04
*   Result of running `:! echo $PATH`, or `! echo %PATH%` on Windows, in Tridactyl: `/home/sajay/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin`
*   Unix-like only: result of running `printf '%c\0\0\0{"cmd": "run", "command": "echo $PATH"}' 39 | ~/.local/share/tridactyl/native_main.py` in a terminal: `{"cmd": "run", "content": "/home/sajay/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin\n", "code": 0}`

**Comment by bovine3dom** [comment]
That's weird. Have you followed the steps on https://github.com/tridactyl/tridactyl/issues/1229#issuecomment-458583919 ?

**Comment by sajaysurya** [comment]
guiset seems to be the problem.
once I delete userChrome.css, everything seems to work fine.
If, I enable guiset then I get the same problem back.

**Comment by bovine3dom** [comment]
Should be fixed in Tridactyl version 1.4.7.
